### PR TITLE
Log scheduler/job_queue orchestrator when created

### PR DESCRIPTION
**Pull request by xjules**
**Issue**
Make sure that the info on orchestrator is logged correctly~~and as a debug.~~

**Approach**
Inject logging when creating JobQueue / Scheduler.

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

**Comment by berland** [comment]
This changes the logged string making it harder to search for occurences before and after this commit, but that is maybe ok?

**Comment by xjules** [comment]
> This changes the logged string making it harder to search for occurences before and after this commit, but that is maybe ok?

I'll change `JobQueue` to `job_queue` or you meant something else?

**Comment by berland** [comment]
> I'll change `JobQueue` to `job_queue` 
:+1:

File: src/ert/ensemble_evaluator/_builder/_legacy.py [hunk 1]
```diff
@@ -234,17 +236,14 @@ async def _evaluate_inner(  # pylint: disable=too-many-branches
             )
             result = identifiers.EVTYPE_ENSEMBLE_FAILED
 
-        orchestrator = "scheduler"
```
Comment (berland) [inline src/ert/ensemble_evaluator/_builder/_legacy.py:237]: `Scheduler` vs `scheduler`

File: src/ert/ensemble_evaluator/_builder/_legacy.py [hunk 2]
```diff
@@ -214,6 +215,7 @@ async def _evaluate_inner(  # pylint: disable=too-many-branches
                     ee_token=self._config.token,
                     on_timeout=on_timeout,
                 )
+                scheduler_logger.debug("Experiment ran on ORCHESTRATOR: job_queue")
```
Comment (pinkwah) [inline src/ert/ensemble_evaluator/_builder/_legacy.py:218]: I concur with Håvard, and I also think this should be JobQueue as that's what we've been calling it.

**Comment by pinkwah** [comment]
I think it should be `INFO` btw. This is information that is useful to us logged to the azure server and not debug information. There should only be 1 line per experiment so it's fine.

File: src/ert/ensemble_evaluator/_builder/_legacy.py (hunk 2 above)
Comment (xjules) [reply src/ert/ensemble_evaluator/_builder/_legacy.py:218]: Then we need to change to big-poly workflow, which is fine I guess. Will do.

Comment (berland) [reply src/ert/ensemble_evaluator/_builder/_legacy.py]: Just keep the strings as they were prior to this PR. Not worth the effort to change it.

**Pull request merged**